src/helpers/fullscreen.ts

```diff
@@ -1,30 +1,28 @@
 import screenfull from 'screenfull';
 
-const attachFullscreen = () => {
-  const btn = document.getElementById('fullscreen');
-  const el = document.getElementById('game');
-  if (btn && el) {
-    btn.addEventListener('click', () => {
-      if (screenfull.isEnabled) {
-        screenfull.request(el, { navigationUI: 'hide' });
-      }
+const fullscreenAndLandscape = () => {
+  const el = document.querySelector('canvas#game') as Element;
+  if (screenfull.isEnabled) {
+    screenfull.request(el, { navigationUI: 'hide' });
+  }
 
-      // because firefox doesnt support .lock ?
-      // https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_dom_d_.screenorientation.html#lock
-      // @ts-expect-error until https://github.com/microsoft/TypeScript-DOM-lib-generator/issues/1615 is closed
-      window.screen.orientation.lock('landscape');
+  // next we change screen orientation to landscape!
 
-      const lockOrientation =
-        // @ts-expect-error these are deprecated, but might still work
-        window.screen.lockOrientation ||
-        // @ts-expect-error these are deprecated, but might still work
-        window.screen.mozLockOrientation ||
-        // @ts-expect-error these are deprecated, but might still work
-        window.screen.msLockOrientation;
+  // this definitly works on chrome
+  // because firefox doesnt support .lock ?
+  // https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_dom_d_.screenorientation.html#lock
+  // @ts-expect-error until https://github.com/microsoft/TypeScript-DOM-lib-generator/issues/1615 is closed
+  window.screen.orientation.lock('landscape');
```

Comment:
that's a point I've not tested this on any other browser but Chrome. I have FireFox and Edge I could test on.

Reply:
the part below is suppose to work in firefox and ms browsers?
i tried it in firefox mobile and `window.screen.mozLockOrientation` doesn't seem to work :shrug:

Reply:
It's working nicely in Edge and Brave browser! Feels a bit jittery in Firefox 🤔 not sure if it's because I need to turn hardware acceleration on for it maybe.

Reply:
edge uses chrome technology

brave is chromium based

firefox is piss poor at webgl